docs/changelog/113869.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 113869
+summary: Upgrade protobufer to 3.25.5
+area: HDFS
```

Comment:
Not sure if this is the best area to apply. I've marked this as HDFS, but actually the protobuf is used in many places.

Reply:
hm, it turns out it must be one of:

```
Aggregations, Allocation, Analysis, Application, Audit, Authentication, Authorization, Autoscaling, CAT APIs, CCR, CCS, CRUD, Client, Cluster Coordination, Codec, Data streams, DLM, Discovery-Plugins, Distributed, Downsampling, EQL, ES|QL, Engine, Experiences, Extract&Transform, FIPS, Features, Geo, Graph, Health, Highlighting, ILM+SLM, IdentityProvider, Indices APIs, Inference, Infra/CLI, Infra/Circuit Breakers, Infra/Core, Infra/Logging, Infra/Node Lifecycle, Infra/Plugins, Infra/REST API, Infra/Resiliency, Infra/Scripting, Infra/Settings, Infra/Transport API, Infra/Metrics, Ingest, Ingest Node, Java High Level REST Client, Java Low Level REST Client, License, Logs, Machine Learning, Mapping, Monitoring, Network, Packaging, Percolator, Performance, Query Languages, Ranking, Recovery, Reindex, Relevance, Rollup, SQL, Search, Security, Snapshot/Restore, Stats, Store, Suggesters, Task Management, TLS, Transform, TSDB, Vector Search, Watcher
```

Reply:
You probably want this to be Snapshot/Restore since it's for the hdfs snapshot repository.